service/src/main/scala/za/co/absa/loginsvc/rest/config/ConfigProvider.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.loginsvc.rest.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import pureconfig._
+import pureconfig.generic.auto._
+import pureconfig.module.yaml._
+import za.co.absa.loginsvc.rest.config.actuator.GitConfig
+import za.co.absa.loginsvc.rest.config.auth._
+import za.co.absa.loginsvc.rest.config.validation.ConfigValidationException
+import java.nio.file.Files
+
+@Component
+class ConfigProvider(@Value("service\\src\\main\\resources\\application.yaml") yaml : String, @Value("false") testEnvironment: Boolean) {
```

Comment:
Windows path?

Reply:
NeverMind, I misunderstood.

It's fixed now. Thanks for pointing this out.